.travis.yml

```diff
@@ -67,10 +66,11 @@ jobs:
         - yes | sdkmanager "platforms;${ANDROID_API_LEVEL}" > /dev/null
         - yes | sdkmanager "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" > /dev/null
         - yes | sdkmanager "cmake;${ANDROID_CMAKE_VERSION}" > /dev/null
-      before_script:
-        - cd examples
       script:
+        - cd examples
         - ./scripts/build-android --hostOnly --buildGluecodium
+        - cd ../functional-tests
```

Comment:
-1 please use pushd/popd instead:
```
pushd examples
./scripts...
popd
pushd functional-tests
./scripts...
popd
```
to make reordering easier.

Reply:
Done